cuid2.go

```diff
@@ -116,8 +117,11 @@ func CreateId() string {
 	return defaultInit()
 }
 
-func CreateIdOf(len int) string {
-	return Init(DefaultRandom, DefaultCounter, len, DefaultFingerprint)()
+func CreateIdOf(len int) (string, error) {
+	if len < 2 || len > bigLength {
```

Comment:
Why don't we put it inside even `Init` method, so that it is applicable to every cases?

Reply:
Yup! That sounds good.

Reply:
Done with Changes, Please check it!